alert/queries.sql

```diff
@@ -95,3 +95,45 @@ ON CONFLICT (alert_id)
     WHERE
         alert_data.alert_id = $1;
 
+-- name: NoStepsByService :one
```

Comment:
One downside of how we're using `sqlc` is that all queries share the same namespace; so to make it more manageable:
- maybe add a prefix for where it's used
- some of the existing names were pretty cryptic, let's see if we can make it better
- Consider adding an additional comment for the generated method

```suggestion
-- name: Alert_ServiceEPHasSteps :one
-- Returns true if the Escalation Policy for the provided service has at least one step.
```

Reply:
I updated the alerts package with better naming and comments. Feel free to critique. To keep this PR in scope, I only modified queries under the alert package at this time.